pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -905,6 +913,22 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 			networkNameMap[spec.Networks[idx].Name] = &spec.Networks[idx]
 		}
 
+		if multusDefaultCount > 1 {
```

Comment:
I think you also need to add a validation for pod network.

we should reject a creation of a vm/vmi with pod network and also multus default.

Reply:
You could have multus as default and still have the pod network.

Reply:
I think I see what you mean now, it shouldn't named by default.

Reply:
I actually can't find anything in the code that cares about the pod network's name (I had thought there was some logic there).  Any default logic I could find was based of the interface name being eth0 and that is already handled by the changes.

Reply:
Hi @greg-bock Is not about the name is about the function I think we should not allow this configuration

```yaml
---
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachineInstance
metadata:
  labels:
    special: test
  name: test
spec:
  domain:
    devices:
      interfaces:
      - bridge: {}
        name: default
      - sriov: {}
        name: multus-net
  networks:
  - name: default
    pod: {}
  - multus:
      networkName: sriov
      default: true
    name: multus-net
```

The problem is pod network(eth0) will be connected using the bridge and the multus default (also eth0) will pass the device.

so I think we just need to reject the combination of pod network and default multus network.

Reply:
Pod network would end up being eth1 in that instance iirc, I'll have to go dig back through the code. We should absolutely allow a Multus network as default and still be able to attach the pod network.

Reply:
I see where the current implementation would fail now. I'll add this to the validation webhook and make sure I add an example to docs for non pod default with pod as additional CNI network.